### PR TITLE
Translate pytorch_fairseq_roberta.md

### PR DESCRIPTION
**Pull request by euroversedev**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #56 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
RoBERTa 번역 추가

### CODE REVIEW

**Comment by euroversedev** [comment]
안녕하세요 : ) 유로입니다.
번역하면서 리뷰 받고 싶은 점들을 적어보았습니다.
아래 내용에 대해 더 적절한 단어, 자연스러운 방법 등을 알려주시면 감사하겠습니다!

1. self-supervised pretraining, language masking strategy, next-sentence pretraining objective를
각각 자기지도사전학습, 언어마스킹전략, 다음문장사전학습으로 번역하고 괄호 안에 영문을 적었습니다.
2. downstream task, benchmark를 임의로 음차표기 했습니다.
3. feature를 용어집에서 참고하여 특징으로 번역했습니다.

제 PR을 방문해 주셔서 감사합니다. 좋은 하루 보내세요!

File: pytorch_fairseq_roberta.md [hunk 1]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
-benchmark datasets.
+Bidirectional Encoder Representations from Transformers, [BERT][1]는 텍스트에서 의도적으로 숨겨진 부분을 예측하는 학습에 뛰어난 자기지도사전학습(self-supervised pretraining) 기술입니다. 특히 BERT가 학습한 표현은 다운스트림 태스크에 잘 일반화되는 것으로 나타났으며, BERT가 처음 공개된 2018년에 수많은 자연어처리 벤치마크 데이터셋에 대해 가장 좋은 성능을 기록했습니다.
 
-[RoBERTa][2] builds on BERT's language masking strategy and modifies key
-hyperparameters in BERT, including removing BERT's next-sentence pretraining
-objective, and training with much larger mini-batches and learning rates.
-RoBERTa was also trained on an order of magnitude more data than BERT, for a
-longer amount of time. This allows RoBERTa representations to generalize even
-better to downstream tasks compared to BERT.
+[RoBERTa][2]는 BERT의 언어마스킹전략(language masking strategy)을 기반으로 만들어지며, 다음문장사전학습(next-sentence pretraining objective)을 제거하고 훨씬 더 큰 미니 배치와 학습 속도로 훈련하는 등 주요 하이퍼파라미터를 수정합니다. 또한 RoBERTa는 더 오랜 시간 동안 BERT보다 훨씬 많은 데이터에 대해 학습되었습니다. 이를 통해 RoBERTa의 표현은 BERT보다 다운스트림 태스크에 더 잘 일반화될 수 있습니다.
```
Comment (Woodywarhol9) [inline pytorch_fairseq_roberta.md:23]: `RoBERTa는 BERT의 언어 마스킹 전략(language masking strategy)에 기반하지만 몇 가지 차이점이 존재합니다.` 요런 식으로 첫 문장을 수정하고 후에 다른 점을 따로 제시하면 어떨까요??

File: pytorch_fairseq_roberta.md [hunk 2]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
 
 
-### Requirements
+### 요구 사항
 
-We require a few additional Python dependencies for preprocessing:
+전처리 과정을 위해 추가적인 Python 의존성이 필요합니다.
```
Comment (Woodywarhol9) [inline pytorch_fairseq_roberta.md:28]: '전처리'라는 표현이 뭔가 어색하다고 생각해요! 
`추가적인`부터 문장을 시작해도 괜찮을 거 같습니다!

**Review by Woodywarhol9 — COMMENTED** [review]
몇 가지 수정 사항 제안드립니다!

File: pytorch_fairseq_roberta.md [hunk 3]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
 
 with torch.no_grad():
-    # Encode a pair of sentences and make a prediction
+    # 한 쌍의 문장을 인코딩하고 예측
     tokens = roberta.encode('Roberta is a heavily optimized version of BERT.', 'Roberta is not very optimized.')
     prediction = roberta.predict('mnli', tokens).argmax().item()
     assert prediction == 0  # contradiction
 
-    # Encode another pair of sentences
+    # 다른 문장 쌍을 인코딩하고 예측
```
Comment (spongebob03) [inline pytorch_fairseq_roberta.md:75]: 원문과 다르게 윗부분 주석과 맞추어 번역해주신 점이 좋은거 같네요👍

File: pytorch_fairseq_roberta.md [hunk 4]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
-##### Apply Byte-Pair Encoding (BPE) to input text
+##### 입력 텍스트에 Byte-Pair Encoding (BPE) 적용하기
 ```python
 tokens = roberta.encode('Hello world!')
 assert tokens.tolist() == [0, 31414, 232, 328, 2]
 assert roberta.decode(tokens) == 'Hello world!'
 ```
 
-##### Extract features from RoBERTa
+##### RoBERTa에서 특징 가져오기
```
Comment (spongebob03) [inline pytorch_fairseq_roberta.md:51]: 가져오기는 Load도 가져오기로 많이 쓰여서 Extract - '추출'은 어떤지 제안드립니다..!

File: pytorch_fairseq_roberta.md [hunk 5]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
 
-    # Encode another pair of sentences
+    # 다른 문장 쌍을 인코딩하고 예측
     tokens = roberta.encode('Roberta is a heavily optimized version of BERT.', 'Roberta is based on BERT.')
     prediction = roberta.predict('mnli', tokens).argmax().item()
     assert prediction == 2  # entailment
 ```
 
-##### Register a new (randomly initialized) classification head
+##### 새로운 분류 헤드 적용하기
```
Comment (zerojin91) [inline pytorch_fairseq_roberta.md:81]: 헤드라는 표현을 레이어 혹은 분류층으로 다 바꾸는 어떨까요??

**Review by zerojin91 — COMMENTED** [review]
다른 내용들은 매끄럽게 번역된것 같습니다!
고생하셨습니다 :)

**Comment by codingbowoo** [comment]
@euroversedev 팀원분들 코멘트들 반영해주시면 리뷰 및 머지 진행하겠습니다.

File: pytorch_fairseq_roberta.md [hunk 6]
```diff
@@ -16,86 +16,76 @@ demo-model-link: https://huggingface.co/spaces/pytorch/RoBERTa
...
-### Model Description
+### 모델 설명
 
-Bidirectional Encoder Representations from Transformers, or [BERT][1], is a
-revolutionary self-supervised pretraining technique that learns to predict
-intentionally hidden (masked) sections of text. Crucially, the representations
-learned by BERT have been shown to generalize well to downstream tasks, and when
-BERT was first released in 2018 it achieved state-of-the-art results on many NLP
-benchmark datasets.
+Bidirectional Encoder Representations from Transformers, [BERT][1]는 텍스트에서 의도적으로 숨겨진 부분을 예측하는 학습에 뛰어난 자기지도사전학습(self-supervised pretraining) 기술입니다. 특히 BERT가 학습한 표현은 다운스트림 태스크에 잘 일반화되는 것으로 나타났으며, BERT가 처음 공개된 2018년에 수많은 자연어처리 벤치마크 데이터셋에 대해 가장 좋은 성능을 기록했습니다.
```
[the remark this answers is not in the document]
Comment (euroversedev) [reply pytorch_fairseq_roberta.md]: 재성님 덕분에 많은 부분 수정할 수 있었습니다. 감사합니다 😊😁

File: pytorch_fairseq_roberta.md (hunk 3 above)
Comment (euroversedev) [reply pytorch_fairseq_roberta.md:75]: 감사합니다! 오늘 만나뵈어 좋았습니다👍
SoExcited~GIF

File: pytorch_fairseq_roberta.md (hunk 5 above)
Comment (euroversedev) [reply pytorch_fairseq_roberta.md]: 생각해보지 못했던 내용인데 디테일한 리뷰 감사합니다 😁
HeavenlyJoyJerkinsIAmSoExcitedGIF

**Comment by euroversedev** [comment]
@codingbowoo 코멘트 반영 완료했습니다!

**Review by Taeyoung96 — APPROVED** [review]
확인했습니다! 깔끔한 번역 감사드립니다. 😄

**Review by codingbowoo — APPROVED** [review]
써머리 부분도 번역해주시는 센스~
LGTM! 기여에 감사드립니다!
